src/CommunityToolkit.Maui.UnitTests/Converters/BaseConverterTests.cs

```diff
@@ -1,14 +1,14 @@
 ﻿using System.Globalization;
+
 using CommunityToolkit.Maui.Converters;
+
 using Xunit;
 
 namespace CommunityToolkit.Maui.UnitTests.Converters;
 
-public abstract class BaseOneWayConverterTest<TConverter> : ConverterTest<TConverter> where TConverter : ICommunityToolkitValueConverter, new()
-{
+public abstract class BaseOneWayConverterTest<TConverter> : ConverterTest<TConverter> where TConverter : ICommunityToolkitValueConverter, new() {
```

Comment:
looks like your VS is overriding our editorconfig rules, can you fix it? I believe if you run the `dotnet format` command on your terminal it will fix those issues

Reply:
You mean like forms, like indentation and stuff

Reply:
Or run that command in the same folder

Reply:
Well visual studio has a terminal

Reply:
like your `{` is in the same line as the method/class declaration

```cs
// your code is this way
public clas Bla {

    public void Do() {
    }
}

// we like this way
public clas Bla 
{

    public void Do() 
    {
    }
}

Reply:
sorry about that